backend/src/main/java/gov/cdc/usds/simplereport/db/model/PhoneNumber.java

```diff
@@ -0,0 +1,102 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.i18n.phonenumbers.NumberParseException;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PhoneType;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.hibernate.annotations.Type;
+
+@Entity
+public class PhoneNumber extends AuditedEntity {
+  @ManyToOne(optional = false)
```

Comment:
this should be `@ManyToMany` right? multiple people can share a phone number

Reply:
That's a good question. @nullflux and I went back and forth on this. We settled on many-to-one but there are definitely tradeoffs. @nullflux do you have anything to add about why we chose this approach?

FWIW, given that there's no unique constraint on `(type, number)` there's currently nothing stopping multiple people from sharing would be the "same" phone number IRL. Each association would just get have own row.

Reply:
oh, you're right, i didn't realize there wasn't a unique constraint on (type, number). As long as there's nothing stopping multiple people from sharing a phone number (and it seems like there isn't), I'm okay!